src/prefect/workers/base.py

```diff
@@ -1064,6 +1069,14 @@ def _emit_flow_run_executed_event(
             follows=submitted_event,
         )
 
+    def _emit_worker_heartbeat_event(self, heartbeat_origin: str) -> Event:
+        return emit_event(
+            "prefect.worker.heartbeat",
```

Comment:
I wonder if we want to have two events, like ‘prefect.worker.flow-run.heartbeat’ and ‘prefect.worker.cancellation.heartbeat’. As it is we’re going to get two nearly identical events and it _might_ be difficult to build an automation around it, or at least it could be counterintuitive since we’d expect two of these events every 60s or whatever.

Reply:
word